api/Document.json

```diff
@@ -4210,6 +4210,92 @@
           }
         }
       },
+      "fullscreenchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenchange_event",
+          "description": "<code>fullscreenchanged</code> event"
```

Comment:
```suggestion
          "description": "<code>fullscreenchanged</code> event",
```

Reply:
Also I think this should be `<code>fullscreenchange</code>`, not `<code>fullscreenchanged</code>`.